addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -23,9 +51,14 @@
                 <description>Network address of the Chromecast device.</description>
                 <required>true</required>
             </parameter>
+            <parameter name="port" type="integer">
+                <context>network-port</context>
+                <label>Network Port</label>
+                <description>Network port of the Chromecast device.</description>
+                <required>true</required>
```

Comment:
This would almost always be discovered but added a default if created manually (same a milight binding where it is required but a default is provided).